.github/workflows/knapsack.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       RAILS_ENV: test
       DATABASE_URL: "postgres://rails:password@localhost:5432/rails_test"
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
```

Comment:
not strictly related to ruby 3.2 - newer github actions that don't have node warnings